tests/testthat/test-class-workbook.R

```diff
@@ -72,10 +65,20 @@ test_that("wb_set_col_widths", {
     "<col min=\"18\" max=\"16384\" width=\"8.88671875\" style=\"16\"/>"
   )
 
-  expect_silent(wb$set_col_widths(cols = 19, width = 9))
+  expect_silent(wb$set_col_widths(cols = 19, widths = 9))
 
 })
 
+test_that("set_col_widths errors when inconsistent lengths are supplied", {
+  wb <- wbWorkbook$new()
+  wb$add_worksheet("test")
+
+  expect_error(wb$set_col_widths(cols = c(1, 2, 3), widths = c(2, 3)), "compatible length")
+  expect_error(wb$set_col_widths(cols = "Y", widths = 1:2))
+  expect_error(wb$set_col_widths("test", cols = "Y", hidden = 1:2))
+  expect_error(wb$set_col_widths(cols = c("Y", "Z"), hidden = 1))
+})
```

Comment:
I‘m already away from my computers, but what precisely does hidden > 2 do? And why should a length of 1 not be sufficient? From the last example above I’d guess that columns Y and Z should be hidden. Am I wrong?

Reply:
> hidden (Hidden Columns) 
> Flag indicating if the affected column(s) are hidden on this worksheet.
> The possible values for this attribute are defined by the W3C XML Schema boolean datatype.

From the docs. Either 0/1 or `true`/`false`.